warehousing-alternatives-compare-table.md

```diff
@@ -0,0 +1,18 @@
+## Differences between Cloudant to Db2 data flow solutions
+
+### Source options from Cloudant:
+
+technology | available document load endpoints | notes
+--- | --- | --- 
+Apache Bahir Spark connector | `/{db}/_changes`, `/{db}/_all_docs`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | -
+IBM Cloud Functions | `/{db}/_changes`, `/{db}/_design/{design_doc}/_view/{view_name}`, `/{db}/_design/{design_doc}/_search/{search_index_name}`, `/{db}/_find` | - 
+Node-RED | `/{db}/_design/{design_doc}/_search/{search_index_name}` | -
+
+### Sink options to Db2:
+
+technology | load to Db2 notes
+--- | --- 
+Apache Bahir Spark connector | options available to create and overwrite Db2 table
+IBM Cloud Functions | bindings available to execute Db2 SQL statements
+Node-RED | IBM Bluemix starter app built-in packages do not support executing SQL statements
```

Comment:
SQL operations: `INSERT`
Notes: The IBM Bluemix starter app built-in packages do not support additional SQL operations, but additional nodes may be available to be added.

I think that's correct isn't it for the notes?

Reply:
Yes, how about for the last part: `...but additional nodes are available in the npm package manager.`

Reference: https://nodered.org/docs/getting-started/adding-nodes